include/client.h

```diff
@@ -1026,6 +1051,132 @@ class Client
         */
         void save(std::string address);
 
+        /*!
+        *   \brief Appends a dataset to the aggregation list
+        *   \details When appending a dataset to an aggregation list,
+        *            the list will automatically be created if it does not
+        *            exist (i.e. this is the first entry in the list).
+        *            Aggregation lists work by referencing the dataset
+        *            by storing its key, so appending a dataset
+        *            to an aggregation list does not create a copy of the
+        *            dataset.  Also, for this reason, the dataset
+        *            must be placed into the database with a separate call
+        *            to put_dataset().
+        *   \param list_name The name of the aggregation list
+        *   \param dataset The DataSet to append
+        *   \throw SmartRedis::Exception if the command fails
+        */
+        void append_to_list(const std::string& list_name,
+                            const DataSet& dataset);
+
+        /*!
+        *   \brief Delete an aggregation list
+        *   \details The key used to locate the aggregation list to be
+        *            deleted may be formed by applying a prefix to the
+        *            supplied name. See set_data_source()
+        *            and use_list_ensemble_prefix() for more details.
+        *   \param list_name The name of the aggregation list
+        *   \throw SmartRedis::Exception if the command fails
+        */
+        void delete_list(const std::string& list_name);
+
+        /*!
+        *   \brief Copy an aggregation list
+        *   \details The source and destination aggregation list keys used to
+        *            locate and store the aggregation list may be formed by
+        *            applying prefixes to the supplied src_name and dest_name.
+        *            See set_data_source() and use_list_ensemble_prefix()
+        *            for more details.
+        *   \param src_name The source list name
+        *   \param dest_name The destination list name
+        *   \throw SmartRedis::Exception if the command fails
+        */
+        void copy_list(const std::string& src_name,
+                       const std::string& dest_name);
+
+        /*!
+        *   \brief Rename an aggregation list
+        *   \details The old and new aggregation list key used to find and
+        *            relocate the list may be formed by applying prefixes to
+        *            the supplied old_name and new_name. See set_data_source()
+        *            and use_list_ensemble_prefix() for more details.
+        *   \param old_name The old list name
+        *   \param new_name The new list name
+        *   \throw SmartRedis::Exception if the command fails
+        */
+        void rename_list(const std::string& src_name,
+                         const std::string& dest_name);
+
+        /*!
+        *   \brief Get the length of an aggregation list
```

Comment:
Rather than repeat the name of the function, how about something like "Get the number of elements in an aggregation list"